docs/root/configuration/overview/extension.rst

```diff
@@ -40,17 +40,35 @@ management server<xds_protocol>` using :ref:`ExtensionConfiguration discovery
 service<envoy_v3_api_file_envoy/service/extension/v3/config_discovery.proto>`.
 The name field in the extension configuration acts as the resource identifier.
 
+Listener filters
+^^^^^^^^^^^^^^^^
+
+For Listener filters, the discovery service configuration is: :ref:`dynamic listener filter
+re-configuration<envoy_v3_api_field_config.listener.v3.ListenerFilter.config_discovery>`.
+The dynamic listener filter config is only supported in TCP listeners.
+If the dynamic configuration is missing, the connection will be rejected until a valid config is updated.
+
+Network filters
+^^^^^^^^^^^^^^^
+
+For downstream network filters, the discovery service configuration is: :ref:`dynamic filter
+re-configuration<envoy_v3_api_field_config.listener.v3.Filter.config_discovery>`.
+If the dynamic configuration is missing, the connection will be rejected until a valid config is updated.
```

Comment:
Please clarify when the new network filter configuration will take into effect.
IIUC this will only be for new connections, and won't be applied to ongoing (long-living) connections.

Reply:
done, thanks